hackathon_site/dashboard/frontend/src/pages/TeamDetail/TeamDetail.module.scss

```diff
@@ -0,0 +1,34 @@
+.container {
+    display: flex;
+    flex-direction: column;
+    gap: 50px;
+}
+
+.left,
+.right {
+    display: flex;
+    flex-direction: column;
+    flex: 1;
+    gap: 10px;
+}
+
+.tableContainer {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+    gap: 14px;
+    padding: 0;
+
+    .tableTitle {
+        text-align: left;
+    }
+
+    th,
+    tr {
+        white-space: nowrap;
+    }
+
+    .checkBox {
+        color: #2b7bbc;
```

Comment:
.checkBox {
        margin-left: -15px;
        color: #2b7bbc;
    }

Add a margin so the title of the table will be on the same line as the checkbox.

Before:
<img width="618" alt="CleanShot 2022-05-29 at 16 00 07@2x" src="https://user-images.githubusercontent.com/36619969/170889253-c110c03e-aa32-4a04-863f-3333f79f2252.png">

After:
<img width="623" alt="CleanShot 2022-05-29 at 15 59 51@2x" src="https://user-images.githubusercontent.com/36619969/170889255-729afdbf-3302-400b-a32d-8b8315045fca.png">

Hope this will work on your divide. If not, we might need to think other approach.

Reply:
Make sure you try this on different screen sizes, but if you want to remove the padding and margin, setting it to 0 in the css should work.